### PR TITLE
Integration Test : Default ListenPort assignment in sever template 

### PR DESCRIPTION
**Pull request by anpanigr**
Add integration test to verify the default ListenPort ( plain text and SSL) when the ListenPort attribute is not set explicitly in WLST offline script or WDT model file

The PR covers 3 usecases 
 (a) No explicit port for Listen/SSL Port  in model/py script   
         (ItIntrospectVersion.java  and ItMiiCustomSslStore.java, ItT3Channel.java,ItTwoDomainsLoadBalancers.java )
 (b) Explicit port with on-prem  default Listen (7001)  and  SSL (7002 ) port  (ProductionModeSecure.java)
 (c) Explicit non on-prem default port such as  eg. 8001/8002 (already covered)

In case (a)  The runtime will be 7100/8100   (b) runtime will be 7001/7002. (c) runtime will be 8001/8002

Ref :  https://confluence.oraclecorp.com/confluence/display/ASDevWLS/Operator+and+WebLogic+network+ports+resolution

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4761/ (parallel)
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/4762/ (sequential)

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItProductionSecureMode.java [hunk 1]
```diff
@@ -200,20 +203,37 @@ public void beforeEach() {
...
+    int defaultAdminPort = getServiceNodePort(
          domainNamespace, getExternalServicePodName(adminServerPodName), "default-admin");
-    assertTrue(sslNodePort != -1,
+    assertTrue(defaultAdminPort != -1,
           "Could not get the default-admin external service node port");    
-    logger.info("Found the administration service nodePort {0}", sslNodePort);
+    logger.info("Found the administration service nodePort {0}", defaultAdminPort);
+
+    // Here the SSL port is explicitly set to 7002 (on-prem default) in
+    // in ServerTemplate section on toplogy file. Here the generated
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItProductionSecureMode.java:219]: typo here: topology

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItT3Channel.java [hunk 2]
```diff
@@ -290,6 +293,28 @@ public void testAdminServerT3Channel() {
...
+
+    // Since the ListenPort is not set explicitly on ServerTemplate 
+    // in wlst offline script wlst-create-domain-onpv.py, the default 
+    // ListenPort on each manged server is set to 7100. This can be confirmed
+    // by intorospecting the corresponding cluster service port (default)
+    int servicePort = assertDoesNotThrow(()
+        -> getServicePort(domainNamespace, 
+              domainUid + "-cluster-" + clusterName, "default"),
+              "Getting Cluster Service default port failed");
+    assertEquals(7100, servicePort, "Default Service Port is not set to 7100");
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItT3Channel.java:305]: can you create a constant for 7100 here?

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItT3Channel.java [hunk 3]
```diff
@@ -290,6 +293,28 @@ public void testAdminServerT3Channel() {
...
+        -> getServicePort(domainNamespace, 
+              domainUid + "-cluster-" + clusterName, "ms-nap"),
+              "Getting Cluster Service nap port failed");
+    assertEquals(8011, napPort, "Nap Service Port is not set to 8011");
+
+    servicePort = assertDoesNotThrow(()
+        -> getServicePort(domainNamespace, 
+              domainUid + "-managed-server1", "default"),
+              "Getting Managed Server Service default port failed");
+    assertEquals(7100, servicePort, "Default Managed Service Port is not 7100");
```
Comment (xiancao) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItT3Channel.java:316]: can you create a constant variable for 7100?

File: integration-tests/src/test/resources/python-scripts/wlst-create-domain-onpv.py [hunk 4]
```diff
@@ -50,8 +50,8 @@ def create_domain():
   st=create(template_name, 'ServerTemplate')
   print('Done creating server template: %s' % template_name)
   cd('/ServerTemplates/%s' % template_name)
-  print('Set managed server port in template')
-  cmo.setListenPort(int(managed_server_port))
+  print('(SKIP) Set managed server port in template')
+  #cmo.setListenPort(int(managed_server_port))
```
Comment (xiancao) [inline integration-tests/src/test/resources/python-scripts/wlst-create-domain-onpv.py:54]: remove the commented out lines.

**Review by xiancao — CHANGES_REQUESTED** [review]

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItProductionSecureMode.java (hunk 1 above)
Comment (anpanigr) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItProductionSecureMode.java:219]: done

File: integration-tests/src/test/resources/python-scripts/wlst-create-domain-onpv.py (hunk 4 above)
Comment (anpanigr) [reply integration-tests/src/test/resources/python-scripts/wlst-create-domain-onpv.py:54]: Added one more line of comment to describe the reason
----
print('(SKIP) Set managed server port in template')
#If no explicit listenport is set, operator infra will pick 7100
#cmo.setListenPort(int(managed_server_port))
----

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItT3Channel.java (hunk 2 above)
Comment (anpanigr) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItT3Channel.java:305]: done

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItT3Channel.java (hunk 3 above)
Comment (anpanigr) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItT3Channel.java:316]: done

**Comment by hzhao-github** [comment]
LG

**Review by hzhao-github — APPROVED** [review]

**Review by xiancao — APPROVED** [review]

**Pull request merged**